### DISCUSSION TITLE
What does "prelude" mean?

### DISCUSSION THREAD

**Discussion by williamyeny** [csstree/csstree#168] [Q&A] [answered]
Apologies for the basic question, but I searched around and I'm still confused what "prelude" means, exactly. Is there a definition somewhere I can read up on? Or an example of a prelude? Thanks!

**Comment by lahmatiy** [comment, marked as answer]
There is no definition, but [CSS Syntax](https://www.w3.org/TR/css-syntax-3/) spec is using it to name a part of a rule. [For instance](https://www.w3.org/TR/css-syntax-3/#parsing):

> **at-rule**
An at-rule has a name, a prelude consisting of a list of component values, and an optional block consisting of a simple {} block.
>
> **qualified rule**
A qualified rule has a prelude consisting of a list of component values, and a block consisting of a simple {} block.
